Runtime/Model/BacktraceConfiguration.cs

```diff
@@ -222,9 +229,28 @@ public class BacktraceConfiguration : ScriptableObject
         [Tooltip("This specifies in which order records are sent to the Backtrace server.")]
         public RetryOrder RetryOrder;
 
+        /// <summary>
+        /// Get full paths to attachments added by client
+        /// </summary>
+        /// <returns>List of absolute path to attachments</returns>
+        public List<string> GetAttachmentPaths()
+        {
+            var result = new List<string>();
+            if (AttachmentPaths == null || AttachmentPaths.Length == 0)
+            {
+                return result;
+            }
+
+            foreach (var path in AttachmentPaths)
+            {
+                result.Add(ClientPathHelper.GetFulLPath(path));
+            }

```

Comment:
@konraddysput  Corner case you should handle (sorry if I missed it elsewhere:  Check for null / empty items in the AttachmentPaths list.  Given the user can create that list + amend it via the inspector, they could accidentally leave one of the items empty.  I dont think this will throw a null reference exception in that case because I am pretty sure the list would just contain an empty string, but might be worth handling (or at least testing).  Let me know what you think

Might make sense to put this in the Initialize method when you assign the value in the config to the client's AttachmentPath property instead.  Ill leave that up to you.

Reply:
It's handled in different places. User attachment never will be null. Reference:
- user attachments: https://github.com/backtrace-labs/backtrace-unity/blob/753f3a044b7b86386c419adcb76980a75a6b7f06/Runtime/Model/BacktraceConfiguration.cs#L239
- report attachments: https://github.com/backtrace-labs/backtrace-unity/blob/753f3a044b7b86386c419adcb76980a75a6b7f06/Runtime/Model/BacktraceReport.cs#L105

If the list is empty in both examples - it's fine. If list has empty values it's also fine - we will check if file exists before uploading it to Backtrace. We wont want to check if file exists here in this function because user might want to create them after backtrace client initialization.

Reply:
Ok great, makes sense.  Thanks

Reply:
@Dautery91 it won't cost us anything to anyway prevents from setting empty strings in editor. I will add anyway a check on the startup. Thanks for pointing this out.